pepper-apis/dsm-core/src/main/java/org/broadinstitute/lddp/email/SendGridClient.java

```diff
@@ -20,29 +20,27 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Used for integrating with SendGrid...
+ * SendGridClient is a legacy wrapper for using SendGrid, carried over from
+ * the legacy DDP repo, and used exclusively by DSM.
  */
 public class SendGridClient {
 
     private static final Logger logger = LoggerFactory.getLogger(SendGridClient.class);
 
     private static final String LOG_PREFIX = "SendGrid";
+    private final SendGrid sendGrid;
 
-    private String sendGridKey;
-    private JsonObject settings = null;
-
-    public void configure(String sendGridKey, JsonObject settings) {
-        this.sendGridKey = sendGridKey;
-        this.settings = settings;
+    public SendGridClient(SendGrid sendGrid) {
+        this.sendGrid = sendGrid;
     }
 
     /**
      * Used to send emails using SendGrid templates.
      */
-    public void sendSingleEmail(@NonNull String sendGridTemplate, @NonNull Recipient recipient) {
+    public void sendSingleEmail(@NonNull String sendGridTemplate, @NonNull Recipient recipient, JsonObject settings) {
```

Comment:
Since JsonObject does not tell the reader anything about the expected content, a JavaDoc explanation would be helpful. Also, right below this, ArrrayList on the left hand side should be a List.

Reply:
I think the only thing being pulled out of `settings` is the from email and the from name.  I can create a separate class so this is more clear.